sdcm/cluster_k8s/__init__.py

```diff
@@ -488,7 +488,8 @@ def get_operator_image(self, repo: str = '', chart_version: str = '') -> str:
             f"show chart {chart_name} --devel --repo {repo} --version {chart_version}")
         for line in chart_info.split("\n"):
             if line.startswith("appVersion:"):
-                return f"scylladb/scylla-operator:{line.split(':')[-1].strip()}"
+                found_app_version = line.split(':')[-1].strip().replace('"', '').replace("'", "")
```

Comment:
Could you please add comment with example appVersion we are parsing here

Reply:
Done

Reply:
What do you think about changing it to 
```python
found_app_version =line.removeprefix("apiVersion:").strip(" \t\"'")
```
?

Reply:
I don't see any need to change it. Moreover, I get following result using proposed code:

```
appVersion: "1.6
```

and the existing code in PR provides proper results:
```
1.6
```

Reply:
Typo in the suggested code: "apiVersion:" -> "appVersion:" Sorry about that.